notification/app/notification/services/frontend/FrontendAlerts.scala

```diff
@@ -0,0 +1,46 @@
+package notification.services.frontend
+
+import models.{BreakingNewsNotification, NotificationReport}
+import play.api.Logger
+import play.api.libs.json.Json
+import play.api.libs.ws.WSClient
+import play.mvc.Http.Status.CREATED
+import tracking.{TrackingError, TrackerResult, TrackingObserver}
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+import scalaz.\/
+import scalaz.syntax.either._
+
+class FrontendAlerts(config: FrontendAlertsConfig, wsClient: WSClient) extends TrackingObserver {
```

Comment:
i wanted to distinguish notifications being sent to devices and notifications (alerts) about "push" + contract defined by `NotificationSender` is different and should be implemented by components (providers) that are communicating with devices